compiler/pkg/generator/template/client.go.tmpl

```diff
@@ -42,6 +42,9 @@ const maxRetryCount = 12
 const sleepTime = 5
 const ownershipAnnotation string = "Ownership"
 
+// informerResyncPeriod is in second, default value is 10 Hrs(36000 Sec)
+const informerResyncPeriod = 36000
```

Comment:
@LokeshvarKr didnt we say, we will expose this as env varialbles ?

Reply:
yes @xmen4xp, I have exposed env to set resync period and updated the PR. PTAL.